### ISSUE TITLE
Feature Request: Subscribe to playlist without subscribing to channel.

### ISSUE THREAD

**Issue by n8detar** [bbilly1/tubearchivist#118]
I would like the option to subscribe to just a specific playlist or two from a channel and only download those videos. I have many channels that have their main videos and short clips that were taken from those videos and don't want the clips to be downloaded. Being able to just download from a playlist would allow me to do it.

**Comment by bbilly1** [comment]
That was introduced in [v0.0.8](https://github.com/bbilly1/tubearchivist/releases/tag/v0.0.8). This would be a good opportunity to familiarize yourself with the [wiki](https://github.com/bbilly1/tubearchivist/wiki) to get an idea of the current state of affairs.

**Comment by n8detar** [comment]
OK, thanks I assumed it wasn't something you couldn't do because of an error I got. 
After the feature was added I tried to delete the channel after I subscribed to the playlist. If you are subscribed to both the channel and a playlist then remove the channel it seems to leave the playlist on the front end but removes it in the backend. So when you click on the playlist you get this page: 
![image](https://user-images.githubusercontent.com/24966292/147265175-1d124fe9-4518-439a-bc58-a181d15958a7.png)
This made me think you needed the channel to be added if you wanted to subscribe to the playlist, which now thinking about it I should have put in a bug report for this.

**Comment by n8detar** [comment]
> If you are subscribed to both the channel and a playlist then remove the channel it seems to leave the playlist on the front end but removes it in the backend. So when you click on the playlist you get this page:

Actually, you don't need to be subscribed, this happens whether you are subscribed or unsubscribed to either the playlist or the channel or both.

**Comment by n8detar** [comment]
Ah, your right. When the feature was first added I removed the channel, which I realize now I should have just unsubscribed. After I removed the channel and went to the playlist page, I tried clicking on the channel name and it brought me to an error page. This made me think that you had to be subscribed to the channel. Thinking about it now I should have put in a bug report and not have made assumptions. 
Should I open a new issue for the bug I found? Basically, when you remove a channel the playlists remain and if you click on the channel link rather than the playlist link it brings you to an error page.
